packages/amplify_analytics_pinpoint/android/build.gradle

```diff
@@ -60,8 +60,8 @@ dependencies {
     api amplifyCore
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.17.4'
-    implementation 'com.amplifyframework:aws-auth-cognito:1.17.4'
+    implementation 'com.amplifyframework:aws-analytics-pinpoint:1.17.8'
```

Comment:
We have several PRs in flight that are all going to be changing these versions...we should figure out a good way to coordinate this, and also a way to a global variable.

Reply:
Good call out, thanks! I was thinking to define the version in `gradle.properties` to share across android packages, though I couldn't find a good location to hold this file.